### PR TITLE
added italian locale

### PR DESCRIPTION
**Pull request by darmens**

### CODE REVIEW

File: config/locales/it.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+en:
```
Comment (bricesanchez) [inline config/locales/it.yml]: This it the wrong locale key :)

File: config/locales/it.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  refinery:
+    plugins:
+      refinerycms_search:
+        title: Cerca
+        description: Motore di ricerca per RefineryCMS
```
Comment (bricesanchez) [inline config/locales/it.yml]: It's Refinery CMS not RefineryCMS

File: config/locales/it.yml (hunk 1 above)
Comment (darmens) [inline config/locales/it.yml]: damn... my bad!

File: config/locales/it.yml (hunk 2 above)
Comment (darmens) [inline config/locales/it.yml]: ok, fixing it!

**Comment by darmens** [comment]
Now it should be ok.

**Comment by bricesanchez** [comment]
Thank you @darmens :)

**Pull request merged**